code/game/objects/items/flamethrower.dm

```diff
@@ -32,6 +32,7 @@
 /obj/item/flamethrower/ComponentInitialize()
 	. = ..()
 	AddElement(/datum/element/update_icon_updates_onmob)
+	AddComponent(/datum/component/overlay_lighting, light_range, light_power, light_color, lit)
```

Comment:
the default `light_range` for an atom is `0`; this should be hard defined as `1`

Reply:
`light_range` of zero is fine, as by default things do not emit light
I don't mind if we change the default to 1 but then we'll need to update the `light_system` to be no light by default, and change a ton of things to `STATIC_LIGHT`
On light, I set the default as white light, copying the old system's default for when `light_color` was undefined (check `PARSE_LIGHT_COLOR`)